patch-package
--- a/node_modules/nivo/es/components/charts/bubble/BubbleHtmlNode.js
+++ b/node_modules/nivo/es/components/charts/bubble/BubbleHtmlNode.js
@@ -21,6 +21,7 @@ var BubbleHtmlNode = function BubbleHtmlNode(_ref) {
     return React.createElement(
         'div',
         _extends({
+            id: ((node.data && node.data.id) ? node.data.id : node.id).replace(/[^\w]/gi, '-'),
             style: {
                 position: 'absolute',
                 display: 'flex',
--- a/node_modules/nivo/lib/components/charts/bubble/BubbleHtmlNode.js
+++ b/node_modules/nivo/lib/components/charts/bubble/BubbleHtmlNode.js
@@ -32,6 +32,7 @@ var BubbleHtmlNode = function BubbleHtmlNode(_ref) {
     return _react2.default.createElement(
         'div',
         _extends({
+            id: ((node.data && node.data.id) ? node.data.id : node.id).replace(/[^\w]/gi, '-'),
             style: {
                 position: 'absolute',
                 display: 'flex',
