Index: node_modules/nivo/es/components/charts/bubble/BubbleHtmlNode.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- node_modules/nivo/es/components/charts/bubble/BubbleHtmlNode.js	(date 1520458432000)
+++ node_modules/nivo/es/components/charts/bubble/BubbleHtmlNode.js	(revision )
@@ -21,6 +21,7 @@
     return React.createElement(
         'div',
         _extends({
+            id: ((node.data && node.data.id) ? node.data.id : node.id).replace(/[^\w]/gi, '-'),
             style: {
                 position: 'absolute',
                 display: 'flex',
