patch-package
--- a/node_modules/@nivo/circle-packing/cjs/nivo-circle-packing.js
+++ b/node_modules/@nivo/circle-packing/cjs/nivo-circle-packing.js
@@ -191,6 +191,7 @@ var BubbleHtmlNode = function BubbleHtmlNode(_ref) {
     return React__default.createElement(
         'div',
         _extends({
+            id: ((node.data && node.data.id) ? node.data.id : node.id).replace(/[^\w]/gi, '-'),
             style: {
                 position: 'absolute',
                 display: 'flex',
