patch-package
--- a/node_modules/@nivo/treemap/cjs/nivo-treemap.js
+++ b/node_modules/@nivo/treemap/cjs/nivo-treemap.js
@@ -140,6 +140,12 @@ var TreeMapHtmlNode = function TreeMapHtmlNode(_ref) {
     return React__default.createElement(
         'div',
         _extends({
+            id: (node.data && node.data.data && node.data.data.id) ?
+                    (node.data.data.id + "").replace(/[^\w]/gi, '-') :
+                        ( (node.data && node.data.id) ?
+                            (node.data.id + "").replace(/[^\w]/gi, '-') :
+                            (node.id + "").replace(/[^\w]/gi, '-')
+                        ),
             style: {
                 boxSizing: 'border-box',
                 position: 'absolute',
